roadmap.md

```diff
@@ -46,7 +51,7 @@ Of course, these will be made carefully with a migration plan in place!
 
  * [New index model](https://github.com/taskcluster/taskcluster-rfcs/issues/30)
  * [New artifact API](https://github.com/taskcluster/taskcluster-rfcs/issues/7)
- * [Limited redeployability](https://github.com/taskcluster/taskcluster-rfcs/issues/13) ("limited' meaning that it's suitable for creating development and staging environments, but does not support more than one active production environment)
+ * [Make AWS provisioner worker type definitions public](https://bugzilla.mozilla.org/show_bug.cgi?id=1375155)
```

Comment:
Hm, that's true - I was thinking this would affect external workers too, like scriptworker.  I'll remove the line then.